apps/design/backend/src/app.test.ts

```diff
@@ -745,11 +747,16 @@ test.skip('Election package export', async () => {
     electionId,
     workspace,
     electionSerializationFormat: 'vxf',
-    shouldExportAudio: false,
+    shouldExportAudio: true,
   });
 
+  const contents = assertDefined(
+    fileStorageClient.getRawFile(
+      join(electionRecord.orgId, electionPackageFilePath)
+    )
+  );
   const { electionPackage, electionPackageHash } = (
-    await readElectionPackageFromFile(electionPackageFilePath)
+    await readNestedElectionPackageFromBuffer(contents)
```

Comment:
Since this test was disabled, election package export has changed. Instead of exporting `election-package-<hash>.zip` it now exports a zip with 2 zips:
* `election-package-<hash>.zip`
* `ballots-<hash>.zip`